libs/cardano-ledger-test/bench/Bench/Cardano/Ledger/EpochBoundary.hs

```diff
@@ -38,23 +39,31 @@ import Data.Map.Strict (Map)
 import qualified Data.Map.Strict as Map
 import Data.Maybe (fromJust)
 import Data.Proxy
+import qualified Data.Text as T
 import Data.Word (Word64)
-import Test.Cardano.Ledger.EraBuffet (TestCrypto)
 import Test.Cardano.Ledger.Shelley.Rules.IncrementalStake (aggregateUtxoCoinByCredential)
 
-type TestEra = MaryEra TestCrypto
+type TestEra = MaryEra
 
-payCred :: PaymentCredential TestCrypto
-payCred = KeyHashObj (hashKey . VKey $ VerKeyMockDSIGN 0)
+payCred :: PaymentCredential
+payCred = KeyHashObj $ payAddr28 0
+  where
+    payAddr28 :: Int -> KeyHash 'Payment
+    payAddr28 n =
+      let i = n `mod` 10
+          prefix = T.pack (take 6 (cycle (show i)))
+       in KeyHash $
+            fromJust $
+              hashFromTextAsHex
+                (prefix <> "0405060708090a0b0c0d0e0f12131415161718191a1b1c1d1e")
```

Comment:
Marking this as a potential point of interest. I opted to take the same approach as in `TxOut`. Also did the same for `stakeCreds` below.

Reply:
This is too complicated for what it suppose to be: some arbitrary hash

Reply:
I've fixed this with a much simpler approach